auth_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+TEST_AUTH=true CLIENT_ID=$1 CLIENT_SECRET=$2 go test
```

Comment:
This is ok for the time being but I think I can make even the auth test run in automated environment, I will start a PR for this as soon as this is merged

Reply:
Sounds great. Would be curious to see how you make it work

Reply:
I have some ideas, I will let you know the details if my experiment succeed 😃